snowflake/SAStoken.txt

```diff
@@ -0,0 +1,49 @@
+-- creating database and warehouse --
+
+use role sysadmin;
+
+create database if not exists playdb;
+
+create warehouse if not exists playwh with warehouse_size='xsmall' auto_suspend=60 initially_suspended=true;
+
+use schema playdb.public;
+
+-- cloud storage integration --
```

Comment:
I see you chose coperino-pasterino method, but most of the objects below are not necessary. Please remove them

Reply:
And, this comment is not valid at this place. Please update it.